disttask/importinto/scheduler.go

```diff
@@ -179,37 +176,80 @@ func (s *ImportScheduler) OnSubtaskFinished(ctx context.Context, subtaskMetaByte
 	return json.Marshal(subtaskMeta)
 }
 
-// CleanupSubtaskExecEnv implements the Scheduler.CleanupSubtaskExecEnv interface.
-func (s *ImportScheduler) CleanupSubtaskExecEnv(_ context.Context) (err error) {
+func (s *importStepScheduler) CleanupSubtaskExecEnv(_ context.Context) (err error) {
 	s.logger.Info("CleanupSubtaskExecEnv", zap.Any("taskMeta", s.taskMeta))
 	s.importCancel()
 	s.wg.Wait()
 	return s.tableImporter.Close()
 }
 
-// Rollback implements the Scheduler.Rollback interface.
-// TODO: add rollback
-func (s *ImportScheduler) Rollback(context.Context) error {
-	logutil.BgLogger().Info("rollback", zap.Any("taskMeta", s.taskMeta))
+func (s *importStepScheduler) Rollback(context.Context) error {
+	// TODO: add rollback
+	s.logger.Info("rollback", zap.Any("taskMeta", s.taskMeta))
 	return nil
 }
 
+type postStepScheduler struct {
+	scheduler.EmptyScheduler
+	taskID   int64
+	taskMeta *TaskMeta
+	logger   *zap.Logger
+}
+
+var _ scheduler.Scheduler = &postStepScheduler{}
+
+func (p *postStepScheduler) OnSubtaskFinished(ctx context.Context, metaBytes []byte) ([]byte, error) {
+	subtaskMeta := &PostProcessStepMeta{}
+	if err := json.Unmarshal(metaBytes, subtaskMeta); err != nil {
+		return nil, err
+	}
+	if err := postProcess(ctx, p.taskMeta, subtaskMeta, p.logger); err != nil {
+		return nil, err
+	}
+	return metaBytes, nil
+}
+
 func init() {
-	scheduler.RegisterSchedulerConstructor(
-		proto.ImportInto,
+	prepareFn := func(taskID int64, bs []byte, step int64) (*TaskMeta, *zap.Logger, error) {
+		taskMeta := TaskMeta{}
+		if err := json.Unmarshal(bs, &taskMeta); err != nil {
+			return nil, nil, err
+		}
+		logger := logutil.BgLogger().With(
+			zap.String("component", "scheduler"),
+			zap.String("type", proto.ImportInto),
+			zap.Int64("task-id", taskID),
```

Comment:
```suggestion
			zap.Int64("task-ID", taskID),
```

Reply:
it works like a json field name, better leave it lower case